src/PlanView/PlanView.qml

```diff
@@ -184,6 +184,179 @@ Item {
+                                        text:                  modelData && modelData.display_id !== undefined ? modelData.display_id : "Order ID not available"
+                                        wrapMode:              Text.WordWrap
+                                    }
+
+                                    QGCLabel {
+                                        Layout.preferredWidth: parent.width / 4
+                                        text:                  modelData && modelData.requested_ts ? removeMilliseconds(modelData.requested_ts) : "Requested time not available"
+                                        wrapMode:              Text.WordWrap
+                                        
+                                        function removeMilliseconds(dateString) {
+                                            if (!dateString) return "Date not available"
+                                            const formattedDate = dateString.replace(/(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})(\.\d+)?(.*)/, "$1$3")
+                                            return formattedDate
+                                        }
+                                    }
+
+                                    QGCLabel {
+                                        Layout.preferredWidth: parent.width / 4
+                                        text:                  modelData && displayMissionStatus(modelData.mission_plan)
+                                        wrapMode:              Text.WordWrap
+
+                                        function displayMissionStatus(missionPlan) {
+                                            const qcStatus = missionPlan.qc_status
+                                            if (qcStatus) {
+                                                return qcStatus
+                                            }
+                                            const confluenceQcUrl = missionPlan.confluence_qc_url
+                                            if (confluenceQcUrl) {
+                                                return "QC in confluence"
+                                            }
+                                            return "Mission not QCed"
+                                        }
+                                    }
+                                    
+                                    Item {
+                                        Layout.fillHeight:     true
+                                        Layout.preferredWidth: parent.width / 4
+                                    
+                                        QGCButton {
+                                            anchors.centerIn: parent
+                                            id:               missionButton
+                                            text:             qsTr("Select mission")
+                                            visible:          modelData && modelData.mission_plan !== undefined && modelData.mission_plan !== null
```

Comment:
do we need both undefined and null? i dont know

Reply:
True, we only need != undefined